app/src/main/java/com/example/tanya/godeltechchallengeandroid/util/Prefs.kt

```diff
@@ -0,0 +1,17 @@
+package com.example.tanya.godeltechchallengeandroid.util
+
+import android.content.SharedPreferences
+import javax.inject.Inject
+
+const val PREFS_NAME = "godeltechchallengeandroid_prefs"
```

Comment:
It should be private as I can see you created the interface variable isFirstStart.

Reply:
But then it won't be visible for AppModule, will it?

Reply:
@Tanyayasiuk why do you need it be visible in the AppModule? The constant is a private implementation of your preferences class.

Reply:
@vadzim-salavei ok, probably I won't need it when change arch, I don't know yet) but at the moment it is used in @Provides fun provideSharedPreferences(..)